website/content/en/docs/faqs/_index.md

```diff
@@ -4,6 +4,14 @@ linkTitle: FAQ
 weight: 10
 ---
 
+## What are the the differences between Kubebuilder and Operator-SDK?
+
+Kubebuilder and Operator-SDK are integrated projects which both allow you to quickly scaffold an operator project. Both tools make use [controller-runtime][controller-runtime] and the resulting projects will have the same basic layout. For more information see the doc about [what’s in a basic project][kb-doc-what-is-a-basic-project].  
```

Comment:
```suggestion
Kubebuilder and Operator SDK are both projects that allow you to quickly create and manage an operator project. Operator SDK uses kubebuilder under the hood to do so for Go projects, such that the `operator-sdk` CLI tool will work with a project created by `kubebuilder`. Therefore each project makes use of [controller-runtime][controller-runtime] and will have the same [basic layout][kb-doc-what-is-a-basic-project].
```

Reply:
what is the definitive stylization of the name of our project? I've seen Operator-SDK, operator-sdk, and now Operator SDK

Reply:
Its either all lowercase and hyphenated `operator-sdk` (when referring to the binary itself or repo name) or title'd and not hyphenated `Operator SDK` (when formally referring to the Operator SDK project). I personally prefer always using `operator-sdk` to avoid ambiguity.